packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -43,6 +47,12 @@ export default class HostItem extends Component {
     return host && host.power_state === 'Running'
   }
 
+  _isMaintained = createSelector(
+    () => this.props.hvSupportedVersions,
+    () => this.props.item,
+    (supportedVersions, host) => semver.satisfies(host.version, supportedVersions?.[host.productBrand])
```

Comment:
`undefined` shouldn't trigger a warning.

Reply:
It trigger `an error has occurred`